frontend/src/main/java/puf/frisbee/frontend/view/ProfileView.java

```diff
@@ -39,17 +73,33 @@ public void init(ProfileViewModel profileViewModel, ViewHandler viewHandler) {
         this.profileName.setText(this.profileViewModel.getProfileNameLabel());
         this.profileEmail.setText(this.profileViewModel.getProfileEmailLabel());
 
-        this.profileError.textProperty().bind(this.profileViewModel.getProfileErrorlLabelProperty());
+        this.profileError.textProperty().bind(
+                this.profileViewModel.getProfileErrorlLabelProperty());
     }
 
+    /**
+     * This method is called when the cancel button in the profile is clicked.
+     * It triggers the redirect to the start view.
```

Comment:
redirects

Reply:
the redirect ist doch hier Nomen? das wird ja getriggert?